package.json

```diff
@@ -14,12 +14,13 @@
     "lint": "eslint --cache \"**/*.{js,jsx,ts,tsx}\""
   },
   "dependencies": {
-    "@docusaurus/core": "3.1.0",
-    "@docusaurus/module-type-aliases": "3.1.0",
-    "@docusaurus/plugin-client-redirects": "3.1.0",
-    "@docusaurus/preset-classic": "3.1.0",
-    "@docusaurus/tsconfig": "3.1.0",
-    "@docusaurus/types": "^3.1.0",
+    "@docusaurus/core": "3.1.1",
```

Comment:
Do we still need the docusaurus upgrade with the slack link change?

Reply:
My preference would be to keep the version the same for now, and then focus on the version upgrade in a distinct PR if possible.